experimenter/experimenter/features/manifests/nimbus_fml_loader.py

```diff
@@ -14,26 +14,18 @@ class NimbusFmlLoader:
     BASE_PATH = Path(BASE_DIR, "features", "manifests", "apps.yaml")
     MANIFEST_PATH = Path(BASE_DIR, "features", "manifests")
 
-    def __init__(self, application: str, channel: str, file_location=BASE_PATH):
+    def __init__(self, application: str, channel: str):
         self.application: str = (
             application if application in NimbusConstants.Application else None
         )
         self.channel: str = channel if channel in NimbusConstants.Channel else None
-        self.application_data = self._get_application_data(application, file_location)
 
     @staticmethod
-    def _get_application_data(application_name, file_location=BASE_PATH):
-        """Fetch app data from local apps.yaml file to find application names, github
-        repos, fml paths within the repos, and release versions.
-        """
-        if Path.exists(file_location):
-            with open(file_location) as application_yaml_file:
-                file = yaml.safe_load(application_yaml_file.read())
-                if application_name in file:
-                    return file[application_name]
-                else:
-                    return None
+    @cache
+    def create_loader(application: str, channel: str):
```

Comment:
Since this is caching, we should probably name it something like `get_loader_for`.

Reply:
It's really just initializing a new loader here, so I'd like to keep it as create rather than get

Reply:
Yeah `get` is usually what methods like this are called.  Django calls it `get_or_create` to be explicit.  It's not that big a deal.